pkgs/os-specific/linux/trusting-trust/stage0-posix/hex0.nix

```diff
@@ -0,0 +1,14 @@
+{ runBareCommand
+, system
+, bootstrap-seeds-src
+}:
+let
+  throwSystem = throw "Unsupported system: ${system}";
```

Comment:
Would it be better to set `system` properly on the derivations?  The user might have an i686-linux remote builder they can do the build on, for example.  (This would mean another parameter to `runBareCommand`.  At that point it might be worth getting rid of the indirection and calling `derivation` directly — that would make it easier for me to read at least, because I wouldn't have had to look up what `runBareCommand` is in a different file.)

Reply:
What would be the best way to get the `system` value?